### DISCUSSION TITLE
Bug - Art of War book doesn't stack and is bugged

### DISCUSSION THREAD

**Discussion by Myrik25** [FakeFishGames/Barotrauma#13857] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Since last patch, the skill book Art of War doesn't stack and is bugged. 

### Reproduction steps

When I load up my single player game save the error has been there since the patch

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.5.0 (Blood in the Water Update, hotfix 1)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I created [a ticket about this in the issue tracker](https://github.com/FakeFishGames/Barotrauma/issues/13899), we'll look into it!

**Comment by JeroonBosch** [comment]
Hi, we made a mistake and accidentally made the new skill book for the "War Stories" talent of the Security Officer override the original skill book. We're working on a fix bringing back the original "Art of Submarine Warfare" book, separating it from the Security Officer's version (the talent version shouldn't be buyable, as it doesn't work without acquiring it through the talent)

**Comment by Regalis11** [comment]
This has been fixed in our private repo now, coming up in the next unstable build!

**Comment by 3ap** [comment]
@Regalis11 could you re-check it please? It looks like the problem still occurs.
References:
- More technical details: https://steamcommunity.com/app/602960/discussions/0/4413046903951887649/
- First appearance of the issue: https://steamcommunity.com/app/602960/discussions/0/4364627615930993163/
- My experience as a new player :)

**Comment by GreatBlueMoon** [comment]
Also see the issue still happening.

**Reply by Regalis11** [reply]
Yep, it seems the changes we did to address this didn't work that well. The main issue seems to be that most of the time you get less than 1 skill from the book, which gives the impression the book does nothing. There's also the issue that there's essentially two versions of the book: the one you get from the talent (whose effect scales with the number of kills), and the version you can buy from stores that gives a fixed amount of skills. We've implemented another fix for this for the upcoming release. Sorry for the inconvenience!

**Comment by gerakis** [comment]
Can you increase the power of the war stories version? Right now its incredibly disappointing. Its usually less than a single skillpoint.

**Comment by Jasontti** [comment]
@gerakis Currently it seemed to give at least 2 skill points per book

Tested the fix for this and found no issues.
Closing discussion as completed.
